src/openrct2-ui/windows/DemolishRidePrompt.cpp

```diff
@@ -26,7 +26,7 @@
 #define WW 200
 #define WH 100
 
-money32 gDemolishRideCost;
+money32 _demolishRideCost;
```

Comment:
This variable is only needed in this window, right? Rather than exporting it, just make it static.